src/python/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include MANIFEST.in
+include *.pyx
+include *.py
+include *.pxd
```

Comment:
It looks like MANIFEST is missing a newline at the end of the file.

Reply:
Fixed.